client/network/test/src/lib.rs

```diff
@@ -352,7 +356,11 @@ impl<D, B> Peer<D, B> where
 		headers_only: bool,
 		inform_sync_about_new_best_block: bool,
 		announce_block: bool,
-	) -> H256 where F: FnMut(BlockBuilder<Block, PeersFullClient, substrate_test_runtime_client::Backend>) -> Block {
+	) -> H256
+		where
+			F: FnMut(BlockBuilder<Block, PeersFullClient,
+				substrate_test_runtime_client::Backend>) -> Block
```

Comment:
This formatting is a bit weird :\

Reply:
agreed - have put it in vertical arg style.